README.md

```diff
@@ -47,7 +47,7 @@ Installation
 #### Hardware
 
 PiNanas will need a linux-based host, with:
-- 10GB free disk space
+- 30GB free disk space
```

Comment:
Ha ? 10GB e système ne tourne pas ?

Reply:
De mon côté, j'y suis aux portes des 10Gb et aucun fichiers sur Nextcloud / Jellyfin. Les images docker prennent rapidement de l'espace.

Aussi, il me semble pertinent d'indiquer directement un espace disque certes minimum mais permettant d'installer Pinanas "out of the box" sans être déjà limite. Aussi, 30 Gb (et aujourd'hui quel rationnel à acheter, par exemple, une carte SD 10 vs 30 ?), permettra d'anticiper les prochaines features que nous pourront ajouter.

Reply:
Ok merci. Disons 32GB pour coler aux capacités habituelles des cartes SD ?